### PR TITLE
Bump repo-config to v0.6.2

### PR DESCRIPTION
**Pull request by TalweSingh**

### CODE REVIEW

File: pytests/test_microgrid.py [hunk 1]
```diff
@@ -3,22 +3,23 @@
 
 """Tests for the frequenz.api.microgrid package."""
 
-# pylint: disable=import-outside-toplevel
```
Comment (llucax) [inline pytests/test_microgrid.py:6]: Do you know why did you need to move this to every function? Or is it because it is like that in the repo-config template? If that is the case, I think we should update the repo-config template instead.

**Review by llucax — DISMISSED** [review]

File: pytests/test_microgrid.py (hunk 1 above)
Comment (TalweSingh) [reply pytests/test_microgrid.py:6]: It's because of repo-config template, I can create an issue for this.
Also can you update the branch protection rules here as well pls

**Review by tiyash-basu-frequenz — APPROVED** [review]
